core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -128,3 +153,45 @@ final class InstalledIntelliJ(val root: Path, probePaths: IdeProbePaths, config:
     builder.start()
   }
 }
+
+final class LocalIntelliJ(
+    val root: Path,
+    probePaths: IdeProbePaths,
+    config: DriverConfig,
+    val paths: IntelliJPaths,
+    pluginsBackup: Path
+) extends InstalledIntelliJ(root, probePaths, config) {
+  override protected val ideaProperties: Path = root.resolve("bin").resolve("idea.properties")
+
+  private val ideaPropertiesBackup: Option[Path] = if (ideaProperties.isFile) {
+    val tempPath = Files.createTempFile(root, "idea.properties", ".backup")
+    Some(ideaProperties.copyTo(tempPath))
+  } else None
+
+  Files.write(ideaProperties, ideaPropertiesContent.getBytes(), StandardOpenOption.APPEND)
```

Comment:
Don't we have something like `path.write(content, append = true)` or `path.append(content)`? If not, it is worth adding.

As fo the logic, if we *append* our config won't we override existing paths?